classes/template/PKPTemplateManager.inc.php

```diff
@@ -453,7 +453,7 @@ public function getCachedLessFilePath($name) {
 		$cacheDirectory = CacheManager::getFileCachePath();
 		$context = $this->_request->getContext();
 		$contextId = is_a($context, 'Context') ? $context->getId() : 0;
-		return $cacheDirectory . DIRECTORY_SEPARATOR . $contextId . '-' . $name . '.css';
+		return $cacheDirectory . DIRECTORY_SEPARATOR . $contextId . '-' . $name . '-' . crc32( Application::get()->getRequest()->getBaseUrl($context)) . '.css';
```

Comment:
The `clearCssCache()` uses a `glob('*.css')`, so it's compatible.

Reply:
`PKPRequest::getBaseUrl` doesn't take a `Context` parameter.

Reply:
Thanks! That should be a true, but I'll just remove for simplicity :)